app/js/components/JobHistory.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { highlightBlock } from 'highlight.js';
+import { ipcRenderer as ipc } from 'electron';
+
+import JobHistoryData from './JobHistoryData';
+import style from './JobHistory.css';
+
+class JobHistory extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { job: {}, ...props };
+        this.order = ['UUID', 'Completed', 'Error', 'Started', 'Finished', 'stdout', 'stderr'];
+    }
+
+    componentWillMount() {
+        ipc.on('pass-job-data', (event, data) => {
+            this.setState({ job: data });
+        });
```

Comment:
Someday we will be able to just have this in a shared store, but this is a good solution for now!